packages/embark/templates/.gitignore

```diff
@@ -1,5 +1,4 @@
 **/.embark
-**/chains.json
```

Comment:
@iurimatias can you explain the purpose of this change? The reason I ask is that if it's a change that should affect the dapps people create with `embark new|demo` then `packages/embark/templates/dot.gitignore` should be changed, not this one, I think.